src/content/guides/how-to-create-an-ethereum-account/index.md

```diff
@@ -4,33 +4,31 @@ description: A step-by-step guide on Ethereum account creation using a wallet.
 lang: en
 ---
 
-# How to "register" an Ethereum account
+# How to create an Ethereum account
 
-Anyone can create an Ethereum account for free with a specific type of app commonly known as a wallet. Wallets create and secure the keys that let you hold, send and receive crypto. You can also connect to projects on Ethereum that will let you trade NFTs, exchange tokens, access games, and much more.
+Anyone can create an Ethereum account at any time, for free. There are several ways, but the easiest and most common way is to use an app known as a wallet. Wallets create and secure the keys that enable you to use Ethereum. You can use your wallet to send transactions, check your token balances and connect to apps built on Ethereum. Some "web2" apps now allow you to sign in with Ethereum too.
 
-Unlike opening a new account with a company, creating an Ethereum account is done freely, privately and without asking permission. Accounts are controlled by keys that your wallet software helps you create, and are not issued by a third party, nor stored in a central registry.
+Unlike opening a new account with a company, creating an Ethereum account is done freely, privately and without requiring permission. Accounts are controlled by keys that your wallet software helps you create, and are not issued by a third party, nor stored in a central registry.
 
-## Step 1: Browse our list of wallets
+## Step 1: Choose a wallet
 
-A wallet is like an online bank account for Ethereum. There are dozens of different wallets to choose from—mobile, desktop, or even browser extensions. Our curated list of trusted wallets is a good place to start.
+A wallet is an app that helps you manage your Ethereum account. It uses your keys to send and receive transactions and sign in to apps. There are dozens of different wallets to choose from—mobile, desktop, or even browser extensions.
 
 <ButtonLink to="/wallets/find-wallet/">
   Find a wallet
 </ButtonLink>
 
-## Step 2: Choose a wallet that suits your needs
+If you are new, you can select the “New to crypto” filter on the "find a wallet" page to identify wallets that should include all necessary features suitable for beginners. There are also other profile filters to cater to your needs. These are examples of commonly used wallets - you should do your own research before trusting any software.
```

Comment:
IMO this should have a screenshot for clarity, thoughts @konopkja?

Reply:
images are always a good idea